LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2017 Khem Raj
+Copyright (c) 2020 Alistair Francis <alistair.francis@wdc.com>
```

Comment:
I wonder if its better to remove the Copyright section from here completely and add a CONTRIBUTORS file listing names of all contributors